python/pyhail/keytable.py

```diff
@@ -387,3 +387,51 @@ def to_dataframe(self, expand=True, flatten=True):
     def export_mongodb(self, mode='append'):
         (scala_package_object(self.hc.jvm.org.broadinstitute.hail.driver)
          .exportMongoDB(self.hc.jsql_context, self.jkt, mode))
+
+    def explode(self, column_names):
+        """Explode columns of this KeyTable.
+
+        The explode operation unpacks the elements in a column of type ``Array`` or ``Set`` into its own row.
+        If an empty ``Array`` or ``Set`` is exploded, the entire row is removed from the ``KeyTable``.
+
+        **Examples**
+
+        Assume ``kt`` is a :py:class:`.KeyTable` with two columns: c1 and c2.
+        The types of each column are ``String`` and ``Array[Int]``.
```

Comment:
Sorry, I'm not being clear.  I thought your old example was good, I just thought you needed a simpler one, too.  I'd include both!